src/resources/help/en_US/relnotes3.2.2.html

```diff
@@ -40,6 +40,7 @@
   <li>Enhanced build_visit to apply a patch to fix a compile issue with VTK with gcc 10.3 on Ubuntu 21.04.</li>
   <li>Added a Docker file for Ubuntu 21.04.</li>
   <li>Fixed a bug where the test suite files displayed in the dashboard produced by the regression test suite were blank.</li>
+  <li>Enhanced build_visit to apply a patch to fix a compile issue with p7zip with gcc 10.3 on Ubuntu 21.04.</li>
```

Comment:
Is the problem only encountered on Ubuntu? The patch appears to be applied unilaterally

Reply:
It was encountered on Ubuntu 21.04 with gcc 10.3, but I believe the fix is broadly applicable and I tested it on quartz to do a sanity check to see if my belief was valid.